modules/renter/renter.go

```diff
@@ -363,10 +363,10 @@ func validateSiapath(siapath string) error {
 		return ErrEmptyFilename
 	}
 	if siapath == ".." {
-		return errors.New("siapath cannot be ..")
+		return errors.New("siapath cannot be .. ")
```

Comment:
Is there a reason for adding a space here?

Reply:
it gets ride of a warning. there was warning that errors should not end in .. or .

Reply:
let's enclose it in `'..'` instead